examples/06-advanced-cancellable-promise.php

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use M6Web\Tornado\Adapter;
+use M6Web\Tornado\EventLoop;
+
+// Choose your adapter.
+//$eventLoop = new Adapter\Tornado\EventLoop();
+//$eventLoop = new Adapter\Tornado\SynchronousEventLoop();
+$eventLoop = new Adapter\Amp\EventLoop();
```

Comment:
Can you put the first one by default? Else previous implementations won't be tested, and it looks… illogical 😉